src/main/docs/guide/configurations/dataAccess/postgresSupport.adoc

```diff
@@ -0,0 +1,76 @@
+Micronaut supports reactive and non-blocking client to connect to Postgres using https://github.com/reactiverse/reactive-pg-client[reactive-pg-client], allowing  to handle many database connections with a single thread.
+
+== Configuring the Reactive Postgres Client
+
+To configure the Reactive Postgres you should first add `postgres-reactive` module to your classpath:
+
+[source,groovy]
+.build.gradle
+----
+include::test-suite/build.gradle[tags=postgres-reactive-dependencies,indent=0]
+----
+
+In addition, you might also need to add following Vert.x dependencies:
+
+[source,groovy]
+.build.gradle
+----
+include::test-suite/build.gradle[tags=vertx-dependencies,indent=0]
+----
+
+You should then configure the URI or https://reactiverse.io/reactive-pg-client/apidocs/index.html[`PoolOptions`] of the Postgres server you wish to communicate with in `application.yml`:
+
+[source,yaml]
+.application.yml
+----
+reactive:
+	pg.client:
+		port: 5432
+		host: the-host
+		database: the-db
+		user: test
+		password: test
+		maxSize: 5
+----
+
+TIP: You can also connect to Postgres using `connectionUri`.
+
+Once you have the above configuration in place then you can inject instance of the `io.reactiverse.reactivex.pgclient.PgPool` bean. Following is the simplest way to connect, query:
+
+[source,groovy]
+----
+include::{testsuite}/configuration/postgres/reactive/PostgresReactiveSpec.groovy[tags=query,indent=0]
+----
+
+<1> `client` is an instance of the `io.reactiverse.reactivex.pgclient.PgPool` bean.
+
+For more information on running queries on Postgres using the reactive client please read the "Running quries" section in the documentation of https://reactiverse.io/reactive-pg-client/guide/groovy/index.html[reactive-pg-client].
+
+== Using Testcontainers for testing
```

Comment:
I don't think documenting testcontainers usage is appropriate for this section of the docs

Reply:
We are not documenting everything about Testcontainers. This is just a very basic example to show a way for testing reactive Postgres client.

Reply:
@puneetbehl I realize that, however imo that is out of the scope of our documentation